dashboard/urls.py

```diff
@@ -1,10 +1,11 @@
-from django.conf.urls import include
 from django.urls import path
+from django.conf.urls import include, url
 
 from . import views
 
 app_name = "dashboard"
 urlpatterns = [
+    url("ut_admin_staff/", include("admissions.urls")),
```

Comment:
Please add a TODO here for changing this to "path" and using the user_object as a field being passed down from dashboard (which gets it from login) for later sprints.